tests/unittest/compiler/test_fuse_split_cat.py

```diff
@@ -0,0 +1,157 @@
+#  Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import unittest
+
+import torch
+from aitemplate.compiler import compile_model, ops
+
+from aitemplate.compiler.base import Tensor
+from aitemplate.compiler.public import IntImm
+
+from aitemplate.testing import detect_target
+from aitemplate.testing.test_utils import get_random_torch_tensor, graph_has_op
+
+class FuseSplitCatTestCase(unittest.TestCase):
+    def test_fuse_split_cat_rearrange(self):
```

Comment:
We could add a couple of more tests, for example

* make a non-even split sections such as ```[501, 11]```, which would make sure we don't have any mis-alignment issue

* cover cases where the output tensors of split are fed to different ops

* some output tensors of the split op are used multiple times by the same concat op, e.g.
```
x1, x2 = split(t)
y = concat(x2, x1, x2)
```